### DISCUSSION TITLE
Request to put a new lib

### DISCUSSION THREAD

**Discussion by galipmedia** [cables-gl/cables_docs#514] [Feature Requests] [answered]
I would like to request to have https://github.com/Vanilagy/webm-muxer this lib added to the libs available to new ops. I believe it is a more performant and native (chrome only) way to capture video that may perform better than the existing method relying on whammy.js . 

I tried pasting it directly into the top of an op but it threw errors. I have used the lib outside of cables and it works well.

**Comment by steam0r** [comment, marked as answer]
you can upload your own libs to patches and use them in userops like this:

https://cables.gl/docs/5_writing_ops/dev_libraries/dev_libraries

basically upload as you would with any image/file and then select "use library from patch assets..." to pick from the filemanager

**Comment by galipmedia** [comment]
My apologies. I really should've read the documentation!
